BUILD.gn

```diff
@@ -1194,10 +1196,27 @@ template("dist_zip") {
     write_runtime_deps = _runtime_deps_file
   }
 
+  action(_strip_binaries_target) {
+    forward_variables_from(invoker, [ "testonly" ])
+    script = "//electron/script/strip-binaries.py"
+    deps = [
+      ":$_runtime_deps_target",
+    ]
+    args = [
+      "--directory",
+      rebase_path(root_out_dir),
+      "--target-cpu",
+      target_cpu,
+    ]
+    outputs = [
+      "$target_gen_dir/$_strip_binaries_target/.strip",
```

Comment:
i'm not sure where we landed on this, but if you're going to keep this hack in, at least name this file `.strip_never_created` and add a comment.